src/features/vote/ActiveViewDetailsModal.tsx

```diff
@@ -76,12 +79,33 @@ const _ActiveViewDetailsModal: ForwardRefRenderFunction<
   const [numberRevealedAddresses, setNumberRevealedAddresses] = useState(0);
   const [revealPercentage, setRevealPercentage] = useState(NULL_PERCENTAGE);
   const [copySuccess, setCopySuccess] = useState(false);
+  const [backupSeed, setBackupSeed] = useState("");
+
   // Note: because there is dynamic content, this will rebuild the tooltip for addressing the conditional
   // elements on the page. See ReactTooltip docs.
   useEffect(() => {
     ReactTooltip.rebuild();
   });
 
+  useEffect(() => {
+    const commits = localStorage.getItem("backupCommits");
+    if (roundId && proposal && unix && votingAddress && commits && ancData) {
+      try {
+        const parsedJSON = JSON.parse(commits);
+        const uniqueIdentifier = `${proposal}~${unix}~${ancData}`;
+        if (
+          has(parsedJSON, `${votingAddress}.[${roundId}].${uniqueIdentifier}`)
```

Comment:
just fyi you can use arrays to represent your path ie: `[votingAddress,roundId,uniqueIdentifier]`, slightly easier than using a template string here. also if u want to keep the template string i dont think you need the `[]` around roundid